api/src/main/java/marquez/db/Columns.java

```diff
@@ -80,6 +80,9 @@ private Columns() {}
   public static final String FIELD_UUIDS = "field_uuids";
   public static final String LIFECYCLE_STATE = "lifecycle_state";
 
+  /* DATASET SYMLINK ROW COLUMNS */
+  public static final String IS_PRIMARY = "is_primary";
```

Comment:
Is the `is_primary` column used to indicated that it's the _primary_ name of the dataset?

Reply:
Ahh it totally is 👍